panels/site_sections/attractions.py

```diff
@@ -19,6 +19,15 @@ def _attendee_for_badge_num(session, badge_num, options=None):
     return query.first()
 
 
+def _attendee_for_info(session, first_name, last_name, email, zip_code):
+    return session.query(Attendee).filter(
+        func.lower(Attendee.first_name) == first_name.lower(),
+        func.lower(Attendee.last_name) == last_name.lower(),
+        Attendee.normalized_email == Attendee.normalize_email(email),
+        Attendee.zip_code == zip_code
+    ).first()
```

Comment:
We already have this -- it's `session.lookup_attendee` at https://github.com/magfest/ubersystem/blob/master/uber/models/__init__.py#L705. That version also handles duplicate attendees, which as we found can be pretty important. Looks like that version could stand to use normalized_email though.

Reply:
Good catch, I've just made and tested this change.  You're right that version should use ``normalized_email``, so I've made that change at https://github.com/magfest/ubersystem/pull/3059